src/autolabel/tasks/classification.py

```diff
@@ -21,6 +21,11 @@
 
 import json
 
+from langchain.prompts.example_selector import SemanticSimilarityExampleSelector
```

Comment:
probably don't need these imports now?

Reply:
removed these imports in [e5ff12a](https://github.com/refuel-ai/autolabel/pull/561/commits/e5ff12a1a3a674de746ad9f49b55511486c15f9d)